src/NuGet.Core/NuGet.Commands/RestoreCommand/LockFileBuilder.cs

```diff
@@ -283,6 +285,99 @@ public class LockFileBuilder
             return lockFile;
         }
 
+        private void ValidateLockFileLibrary(LockFile lockFile)
```

Comment:
Given that this method's name does not clearly communicate  what it does, consider it to 
`EnsureUniqueLockFileLibraries`

Reply:
I renamed to `GetUniqueLockFileLibraries`